packages/core-app-api/src/app/AppManager.tsx

```diff
@@ -154,7 +154,53 @@ function useConfigLoader(
     };
   }
 
-  const configReader = ConfigReader.fromConfigs(config.value ?? []);
+  let configReader;
+
+  /**
+   * config.value can be undefined or empty. If it's either, don't bother overriding anything.
+   */
+  if (config.value?.length) {
+    configReader = ConfigReader.fromConfigs(config.value);
```

Comment:
No need for this up here, we're always going to set it below anyway

Reply:
Not sure I'm following, we need to create it initially to access the config values before we override them. Without this line, we can't call `configReader.getString('app.baseUrl')` below.

Reply:
Ah derp, of course. Perhaps declare a separate variable for this thought tbh, just to be safe?